lib/agent.js

```diff
@@ -216,9 +211,11 @@ Agent.prototype.start = function start(callback) {
   }
 
   if (!this.config.license_key) {
-    logger.error('A valid account license key cannot be found. ' +
-                 'Has a license key been specified in the agent configuration ' +
-                 'file or via the NEW_RELIC_LICENSE_KEY environment variable?')
+    logger.error(
+      'A valid account license key cannot be found. ' +
+        'Has a license key been specified in the agent configuration ' +
+        'file or via the NEW_RELIC_LICENSE_KEY environment variable?'
```

Comment:
Not sure if its just how these diffs display but i've seen a few times where it appears too much whitespace was inserted for a sort of awkward nesting. 

If we find this happening in reality, we may need to find and tweak a setting.

Reply:
i noticed this too, let me see if i can figure out why

Reply:
apparently it's by design https://github.com/prettier/prettier/issues/2235

Reply:
thanks, i hate it. haha

Reply:
lol. more positives than negatives when switching to prettier though so 🤷🏻‍♂️

Reply:
yeah. what i didn't realize earlier is this is because it is being directly passed into a function. can improve readability by doing an assignment ahead of time as desired and getting less funky formatting.

certainly not a deal breaker. :)